man/systemd-measure.xml

```diff
@@ -104,6 +104,16 @@
 
         <xi:include href="version-info.xml" xpointer="v252"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>pcrpkey</command></term>
+
+        <listitem><para>This commands writes the public key to stdout extracted from either the public key
```

Comment:
"writes to stdout" → "prints"? Also extracting the public key that was given is a bit strange.

"This command prints the public key either given with the --public-key= option, or extracted from the certificate given with --certificate= or the private key given with --private-key." (No oxford comma here because the second or only has joins two clauses.)